src/main/java/frc/trigon/robot/poseestimation/relativerobotposesource/io/RelativeRobotPoseSourceT265IO.java

```diff
@@ -0,0 +1,55 @@
+package frc.trigon.robot.poseestimation.relativerobotposesource.io;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.Timer;
+import frc.trigon.robot.poseestimation.relativerobotposesource.RelativeRobotPoseSourceIO;
+import frc.trigon.robot.poseestimation.relativerobotposesource.RelativeRobotPoseSourceInputsAutoLogged;
+import org.trigon.utilities.JsonHandler;
+
+public class RelativeRobotPoseSourceT265IO extends RelativeRobotPoseSourceIO {
+    private JsonDump jsonDump = new JsonDump();
+    private final String hostname;
+
+    public RelativeRobotPoseSourceT265IO(String hostname) {
+        this.hostname = hostname;
+    }
+
+    @Override
+    protected void updateInputs(RelativeRobotPoseSourceInputsAutoLogged inputs) {
+        updateJsonDump();
+        final Pose2d lastPose = inputs.pose;
+
+        inputs.framesPerSecond = jsonDump.framesPerSecond;
+        inputs.batteryPercentage = jsonDump.BatteryPercentage;
+        inputs.pose = getT265Pose();
+        inputs.lastResultTimestamp = lastPose == inputs.pose ? inputs.lastResultTimestamp : Timer.getTimestamp();
+    }
+
+    private void updateJsonDump() {
+        final String jsonString = NetworkTableInstance.getDefault().getEntry(hostname).getString("");
```

Comment:
this is done wrongly 
check elsewhere how it's supposed to be done (namely, where should the hostname go and where should the entry name go)